Expensify/react-native-live-markdown
# fix: remove console.error to fix crash on iOS

### Details


### Related Issues


https://github.com/Expensify/App/issues/45623

### Manual Tests


1. Run example iOS app
2. Input `[ a ](https://example.com)`

Verify that there's no crash, and the markdown input still applies the style for the text

### Linked PRs


https://github.com/Expensify/expensify-common/pull/773  - Hold until this is merged
Pending: create bump version PR in app repo.


### Evidence

![md-ios](https://github.com/user-attachments/assets/97f7e0ab-d464-42de-ab8f-73d898b13c31)
